reviews-receiver/src/producer.ts

```diff
@@ -0,0 +1,49 @@
+import { IRawReview } from './../../shared-types/types';
```

Comment:
Import from the package not by relative path
```suggestion
import { IRawReview } from 'shared-types';
```